src/app/feature/competition/pages/manage-competitions/manage-competitions.component.scss

```
@@ -0,0 +1,178 @@
+::host {
```

Stylelint (17.6.0)

[error] Unexpected unknown pseudo-element selector "::host" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)